recipes/lhcb-simpletools/build.sh

```diff
@@ -0,0 +1,20 @@
+mkdir build
+cd build
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [ "$(uname)" == "Linux" ]; then
+    # Fix up CMake for using conda's sysroot
+    # See https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html?highlight=cmake#an-aside-on-cmake-and-sysroots
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_TOOLCHAIN_FILE=${RECIPE_DIR}/cross-linux.cmake")
```

Comment:
Is this necessary?

Reply:
I'm not sure. I've started including it by default when writing CMake based builds based on the advice [here](https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html?highlight=cmake#an-aside-on-cmake-and-sysroots).